docs/source/getstarted/installation.md

```diff
@@ -62,5 +62,6 @@ pip install -e .
 ```
 
 ## Optional Dependencies
-Olive has optional dependencies that can be installed to enable additional features. Please refer to [extra dependencies](https://github.com/microsoft/Olive/blob/main/olive/extra_dependencies.json)
-for the list of extras and their dependencies.
+Olive has optional dependencies that can be installed to enable additional features. Please refer to
+[extra dependencies](https://github.com/microsoft/Olive/blob/main/olive/olive_config.json) for the list of extras
```

Comment:
need refer to your private branch. after the PR is merged, update to main.

Reply:
I don't follow your comment. Please explain!

Reply:
before the PR is merged, the main/olive/olive_config.json will not exist in main branch. As the result, the linkcheck would fail.

Reply:
There is link checker in our CI pipeline and it will fail if the link is unreachable.https://github.com/microsoft/Olive/pull/358

Reply:
So, what's the workaround? Split the PR into two submissions? Or merge the PR while overriding/ignoring the lint checker?

Reply:
Previously, we split the PR into two submissions.

Reply:
Done! Removed docs changes from this PR and will follow up with another ones this is merged.